recipe/meta.yaml

```diff
@@ -1,5 +1,5 @@
-{% set build_num = 1 %}
-{% set min_runtime_version = 12 %}
+{% set build_num = 3 %}
+{% set min_runtime_version = 13 %}
```

Comment:
Should be `12` for `gcc<=12` and `13` for `gcc==13`, right @beckermr?

Reply:
Might be a silly question, but why don't we just depend on the package version natively for this?

Reply:
I don't know the answer to your question @isuruf. Sorry.

Reply:
See https://github.com/conda-forge/arrow-cpp-feedstock/issues/771 and https://matrix.to/#/!SOyumkgPRWoXfQYIFH:matrix.org/$1652721454676XdRLJ:gitter.im?via=matrix.org&via=gitter.im&via=one.ems.host

Reply:
You can get those links if you do a git blame on that code and arrive at https://github.com/conda-forge/ctng-compiler-activation-feedstock/pull/75

Reply:
I did actually click through the git blame, but didn't check all PRs (& associated links) that touched it.

In general, it's very surprising to me that (e.g.) using gcc 10 with `libstdcxx-ng>=10` would break in the presence of newer libstdcxx, given how much gcc cares about compatibility.

> Should be `12` for `gcc<=12` and `13` for `gcc==13`, right @beckermr?

That leaves us pretty much no choice but to go to >=13 everywhere, as soon as gcc 13 becomes our default compiler?

Reply:
Or alternatively, we could add a run-constraint on `libstdcxx-ng ==X` for gcc X. That way, the compiler shouldn't pick up incompatible newer symbols.

Reply:
> That leaves us pretty much no choice but to go to >=13 everywhere, as soon as gcc 13 becomes our default compiler?

No. `>=12` for `gcc<=12` and `>=13` for `gcc=13`. GCC-13 unlike GCC-12 did not change the default version of an ELF symbol. (See https://johannst.github.io/notes/development/symbolver.html)

Reply:
I tried to reflect that in a comment. It's hard for me to tell where such default changes happen upstream (though I guess the `_V2::` stuff is an indicator).